test/testdata/config/benchttp.json

```diff
@@ -9,7 +9,11 @@
       "key0": ["val0", "val1"],
       "key1": ["val0"]
     },
-    "timeout": "2s"
+    "timeout": "2s",
+    "body": {
+      "type": "raw",
+      "content": "{\"key0\":\"val0\",\"key1\":\"val1\"}"
```

Comment:
I have searched quite a moment if I could do something to avoid escaping these double quotes here, but I don't think it is easily possible.
Moreover, we want a string here (and not a map for example) in case the user wants to input totally wrong json too, right?

Reply:
JSON syntax requires it. It's fine i think. And yes we'd like to allow the user to pass invalid json !